tests/collections/asr/test_asr_exportables.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import tempfile
+
+import pytest
+from omegaconf import DictConfig
+
+from nemo.collections.asr.modules import ConvASRDecoder, ConvASREncoder
+
+
+class TestExportable:
+    @pytest.mark.unit
```

Comment:
wasn't it the part of the yesterday's PR?

Reply:
Yes it was - not sure why it's coming up here, will check.